packages/reactive_ble_mobile/ios/Classes/ReactiveBle/Central.swift

```diff
@@ -281,6 +288,26 @@ final class Central {
         return peripheral.maximumWriteValueLength(for: type)
     }
 
+    
+    func readRssi(for peripheralId: PeripheralID, completion: @escaping (Failable<Int>) -> Void) throws {
+        let peripheral = try resolve(connected: peripheralId)
+
+        // register task, completion goes straight to the caller
+        readRssiRegistry.registerTask(
+            key: peripheralId,
+            params: .init(),
+            completion: completion
+        )
+
+        // update task to kick things off
```

Comment:
These two comments can be removed as well IMHO, as they don't give information that isn't already clear.